docs/data/rpc/admin-guide/overview.mdx

```diff
@@ -0,0 +1,12 @@
+---
+title: "Overview"
+sidebar_position: 0
+---
+
+Stellar RPC is a lightweight service that provides real-time access to Stellar network data. Much like RPC nodes in other blockchain ecosystems, it allows developers to query the network efficiently.
+
+The RPC service allows you to communicate directly with a Stellar network via a [JSON RPC interface](../README.mdx).
+
+For example, you can build an application and have it [send a transaction](../api-reference/methods/getTransaction.mdx), [get ledger](../api-reference/methods/getLedgerEntries.mdx) and [event data](../api-reference/methods/getEvents.mdx), or [simulate transactions](../api-reference/methods/simulateTransaction.mdx).
+
+Alternatively, you can use one of Soroban's [client SDKs](/docs/tools/sdks/client-sdks), such as the [@stellar/stellar-sdk](/docs/tools/sdks/client-sdks#javascript-sdk), which will need to communicate with an RPC instance to access the network.
```

Comment:
This page is now pretty duplicative of the overview here - https://github.com/stellar/stellar-docs/blob/main/docs/data/rpc/README.mdx 

As I'm looking at this, I'm now seeing that Horizon also has this duplicity; it has an "Overview" at the top level and then an "Overview" within the "Admin Guide", and the content is overlapping/similar there as well. Galexie does not have an "Overview" inside the "Admin Guide". I think we should follow the Galexie pattern here and keep the overview at the top level (so remove this file). I'll also file a separate issue to collapse the 2 in the Horizon section (looks like that Horizon info also needs a general refresh).